Libplanet/Net/Swarm.BlockSync.cs

```diff
@@ -112,15 +117,18 @@ CancellationToken cancellationToken
             {
                 if (BlockDemandTable.Any())
                 {
-                    List<(BoundPeer, IBlockExcerpt)> peersWithExcerpt = BlockDemandTable.Demands
-                        .Select(pair => (pair.Key, (IBlockExcerpt)pair.Value)).ToList();
-                    await ProcessFillBlocksAsync(
-                        peersWithExcerpt,
+                    var largest =
+                        BlockDemandTable.Demands.Values.OrderBy(demand => demand.TotalDifficulty)
+                            .Last();
```

Comment:
IMO, we don't need full ordering.

```suggestion
                        BlockDemandTable.Demands.Values
                        .Aggregate(acc, next => next.TotalDifficulty > acc.TotalDifficulty ? next : acc)
```